scripts/downloads/download_data.py

```diff
@@ -94,43 +119,119 @@ def download_capture_name(save_dir: Path, dataset_name: str, capture_name: str,
     os.remove(download_path)
 
 
-def download_nerfstudio(save_dir: Path, capture_name: str):
-    """Download specific captures for the nerfstudio dataset."""
-    download_capture_name(save_dir, "nerfstudio", capture_name, capture_name_to_file_id=nerfstudio_file_ids)
+@dataclass
+class NerfstudioDownload(DatasetDownload):
+    """Download the nerfstudio dataset."""
+
+    capture_name: NerfstudioCaptureName = "bww_entrance"  # type: ignore
```

Comment:
If you add a `from __future__ import annotations` to the top of this file you should be able to get rid of these `# type: ignore`s (at least for pyright/VS Code)